File: src/providers/gemini.ts
/**
 * Gemini CLI agent configuration
 */

import type { AgentConfig } from "../types.js";

/**
 * Default configuration for Gemini CLI agent via --experimental-acp flag
 */
export const geminiConfig: AgentConfig = {
  command: "npx",
  args: ["@google/gemini-cli", "--experimental-acp"],
  env: {},
};

File: python/src/acp_factory/providers/gemini.py
"""
Gemini CLI agent configuration
"""

from acp_factory.types import AgentConfig

# Default configuration for Gemini CLI agent via --experimental-acp flag
gemini_config = AgentConfig(
    command="npx",
    args=["@google/gemini-cli", "--experimental-acp"],
    env={},
)
